dashboard.py

```diff
@@ -242,12 +244,13 @@ def make_ab_confidence_map(
             "runtime",
         ]
     ]
+
     runtimes = df["runtime"].unique()
-    A = baseline
-    B = next((r for r in runtimes if r != baseline), None)
-    if not B:
```

Comment:
I still think this fix was necessary -- there are lots of ways to get subsets of data that only include the baseline numbers.

Reply:
On this line, you'll always get at most baseline + exactly one other runtime.
My change adds a check for the existence of baseline, but does not remove the check for the other runtime.

Reply:
If you have a subset of data with *only* baseline runtimes, your check still causes a `StopIteration` error.

Reply:
If you have only baseline, then the check 2 lines above `if len(runtimes) < 2` will fail.